tests/test_other.py

```diff
@@ -7943,9 +7943,9 @@ def run(*args):
       run(['-O1'],  9, [], ['waka'],  8095,  2, 12, 10) # noqa
       run(['-O2'],  9, [], ['waka'],  8077,  2, 12, 10) # noqa
       # in -O3, -Os and -Oz we metadce, and they shrink it down to the minimal output we want
-      run(['-O3'],  0, [], [],          61,  0,  1,  1) # noqa
-      run(['-Os'],  0, [], [],          61,  0,  1,  1) # noqa
-      run(['-Oz'],  0, [], [],           8,  0,  0,  0) # noqa
+      run(['-O3'],  0, [], [],          85,  0,  2,  2) # noqa
+      run(['-Os'],  0, [], [],          85,  0,  2,  2) # noqa
+      run(['-Oz'],  0, [], [],          54,  0,  1,  1) # noqa
```

Comment:
I think the code is right, but the results are slightly surprising.

Was `-Oz` was basically empty and incorrect before, I suppose? And what is the cause of the smaller increase in the others?

Reply:
Its looks like its due to the __post_instantiate function being includes. And also the name section is still in there... which is a separate bug that we should look at.  But this change is strictly more correct.

Reply:
I agree this change is more correct, yeah. Would be good to improve things so post-instantiate etc. can also be metadce'd out, so we are comparable to the asm2wasm path (55 bytes vs 85, in -Os/-Oz).